packages/rulesets/src/spectral/az-common.ts

```diff
@@ -228,7 +228,7 @@ const ruleset: any = {
       message: "'{{error}}' is not a known format.",
       severity: "error",
       resolved: false,
-      given: "$..[?(@property === 'format')]^",
```

Comment:
Previous query:
* Find all properties named `format`
* Return their parent object (`^`)

New query:
* Find all objects with a property named named `format` AND
* A property named `type` with value `"boolean" | "integer" | "number" | "string"`
* Don't need to return parent using `^` since we are searching objects, not properties